### ISSUE TITLE
Copy the font configuration file ????

### ISSUE THREAD

**Issue by sdyalor** [ryanoasis/nerd-fonts#402]
Where is that configuration file in the .zip from hack font???

**Comment by ryanoasis** [comment]
What do you mean by configuration file? 

.zip you are referring to the release asset file for hack?

**Comment by sdyalor** [comment]
Ahm. I'v read other readme's and there is a config file for standalone installation but i couldn't find it

**Comment by ryanoasis** [comment]
@sdyalor Sorry but I still do not follow.

When you say config file I'm thinking of https://github.com/ryanoasis/nerd-fonts/blob/master/src/unpatched-fonts/Hack/config.cfg or https://github.com/ryanoasis/nerd-fonts/blob/master/src/unpatched-fonts/Arimo/config.json and those do not have to do with installation.

Can you paste a link to what exact config file you are referring to?

**Comment by hemantsharma1498** [comment]
Hi, 
I've run into the same issue regarding the configuration file in 'config/fontconfig' as stated in Quick Installation (Linux) step 4, in the URL below:
https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Hack#quick-installation

What is the config file that is being referred to? It isn't included in .zip

**Comment by Finii** [comment]
TL;DR: You should follow the Nerd Font [installation hints](https://github.com/ryanoasis/nerd-fonts#font-installation) instead.

> What is the config file that is being referred to? It isn't included in .zip

The readme.md you are seeing there is the 'original Hack' website, and does not (nessesarily) relate at all to the Nerd Font versions of fonts. It is unfortunate that there is no indication on the page, that it belongs to the original (unpatched) font. The reason is the automated readme generation process, where it is just copied over. Sometimes (often) it contains valuable information for a font (maybe not for Hack).

I have no clue what they have in the `fontfonfig` file. Why do you think you need one? Something not working? Then, please, raise a new Issue.

There **is** a Nerd Font `fontconfig` file, but probably you do not want that, because it enabled you to use any (unpatched) font with the Symbols-Only Nerd Font, no patching required.

File: https://github.com/ryanoasis/nerd-fonts/blob/master/10-nerd-font-symbols.conf
Some very (very) rough info on that: https://github.com/ryanoasis/nerd-fonts/wiki/Font-Substitution-&-Fallback

**Comment by Finii** [comment]
> Quick Installation (Linux) step 4

Btw, if you follow that, in step 1 it dowloads the unpatched (original) Hack font :grimacing:

This should probably be cleaned up.
